test/over_react_test/react_util_test.dart

```diff
@@ -785,22 +821,33 @@ main() {
 
           expect(queryAllByTestId(renderedInstance, 'value'), innerNodes);
         });
+      });
 
-        test('expect when no matching element exists', () {
-          var renderedInstance = render(Dom.div()());
+      group('from multiple layers within nested ShadowRoots when `searchInShadowDom` is `true`', () {
```

Comment:
Based on the above comment, maybe there should also be a test where `searchInShadowDom` is `false` and verify it returns empty

Reply:
good point!